src/main/resources/css/common.css

```diff
@@ -165,6 +168,10 @@ span {
         bottom: 0;
         left: 0;
     }
+    .navbar__item {
+        display: flex;
+        flex-direction: column;
+    }
```

Comment:
just to clarify, why do we set `.navbar__item` here to be `display: flex` when on line 80 we are changing it to `display: inline-flex`? From my understanding they serve a similar function, so why do think we need to use inline-flex for devices larger that 1200px?